### ISSUE TITLE
Sentiment Analysis on Twitter Data

### ISSUE THREAD

**Issue by ASLManasa** [abhisheks008/ML-Crate#41]
### ML-Crate Repository (Proposing new issue)
:red_circle: **Project Title** : Sentiment Analysis on Twitter Data
:red_circle: **Aim** :  Analysing the sentiment of the users and creating a prediction model based on the data, which will predict the sentiment of the user.
:red_circle: **Dataset** : https://www.kaggle.com/arkhoshghalb/twitter-sentiment-analysis-hatred-speech
:red_circle: **Approach** :Try to implement more than three machine learning algorithms and make a comparison between them, then conclude about the best fit algorithm to create the model. All the algorithms must be checked through the accuracy scores to determine the best fitted model

Hello, ML-Crate contributors, this issue is only for the contribution purposes and allocated only to the participants of SWOC 2.0 Open Source Program.

**********************************************************************************
### 📍 **Follow the Guidelines to Contribute in the Project :**
* You need to create a separate folder named as the Project Title.
* Inside that folder, there will be four main components.
   * Images - To store the required images.
   * Dataset - To store the dataset or, information/source about the dataset.
   * Model - To store the machine learning model you've created using the dataset.
   * `requirements.txt` - This file will contain the required packages/libraries to run the project in other machines.
* Inside the `Model` folder, the `README.md` file must be filled up properly, with proper visualizations and conclusions.
* A Demo Project has been created to guide you about the whole structure of presenting the Project in this repository. Here's the link of the Demo Project - https://github.com/abhisheks008/ML-Crate/tree/main/Project-Demo-Folder

***********************************************************************
:red_circle::yellow_circle: **Points to Note :**

- The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.
- "Issue Title" and "PR Title should be the same. Include issue number along with it.
- Follow Contributing Guidelines & Code of Conduct before start Contributing.
- This issue is only for 'SWOC' contributors of 'ML-Crate' project.

***********************************************************************
:white_check_mark: **To be Mentioned while taking the issue :**
- Full name : 
- GitHub Profile Link : 
- Participant ID :
- Approach for this Project :
- Are you a participant of SWOC 2.0?
    - [ ] YES
    - [ ]  No

*************************************************************
Happy Contributing 🚀 

All the best. Enjoy your open source journey ahead. 😎

**Comment by Cody-coder017** [comment]
Full name : Venkatakrishnan R
GitHub Profile Link :https://github.com/Cody-coder017
Participant ID : 1104
Approach for this Project : Using NLP to train model on various sentiments
I would like to work on this issue

**Comment by rohansharma4050** [comment]
@Cody-coder017 Issue assigned to you, best of luck.

**Comment by abhisheks008** [comment]
@Cody-coder017 complete the previous issue before starting this.
